hail/python/hail/experimental/datasets.py

```diff
@@ -0,0 +1,294 @@
+
+import json
+import hail as hl
+
+
+def load_dataset(dataset_name,
+                 reference_genome,
+                 config_file='gs://hail-datasets/hail_datasets.config.json'):
+    """Load a Hail-formatted genetic dataset.
+
+    Example
+    -------
+
+    >>> # Load 1000 Genomes chromosome X MatrixTable with GRCh38 coordinates
+    >>> mt_1kg = hl.experimental.load_dataset('1000_genomes_phase3_chrX',   # doctest: +SKIP
+    ...                                       reference_genome='GRCh38')
+
+    >>> # Count rows and columns
+    >>> mt_1kg.count()  # doctest: +SKIP
+
```

Comment:
doctest syntax lets you put output on the next line at the same level of indentation:

```
>>> mt_1kg.count()  # doctest: +SKIP
(3474116, 2504)
```

Would prefer this because it'll format like the rest of our doctest examples (the output will be gray, not black)